src/plugins/utils.ts

```diff
@@ -0,0 +1,15 @@
+export const orderArray = (array: any[]) => {
+  const result: any[] = [];
+  array.forEach((item: any) => {
+    if (!item) {
+      return;
+    }
+    const index = result.findIndex(x => x.metadata.order < item.metadata.order);
```

Comment:
Metadata are optional:
```suggestion
    const index = result.findIndex(x => (x.metadata?.order ?? -Infinity) < (item.metadata?.order ?? -Infinity));
```